docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the RBAC based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same (cluster) role and (cluster) role binding priniciples apply the same way as in Kubernetes.
+
+However, since kcp is based around workspaces and workspaces are forming a tenancy boundary,
+RBAC rules are always declared within the context of a concrete workspace.
+These rules apply within the workspace they were declared and do not apply in other workspaces.
+The type of the workspace doesn't matter, it can either be the enduser, organizational, or a root workspace.
+The exception are system namespaces. These can only be accessed by shard-local admins.
+
+# Authorizers
+
+## Workspace content authorizer
+
+As described above, RBAC rules apply for resources within a workspaces only.
```

Comment:
see comment above. I think this statement is not correct, I mean the "only" in the sentence. They apply locally in that workspace as defined by Kube. In addition, they apply .... workspace content, type authorization, maybe more

Reply:
changed to 

> As described above, RBAC rules apply for resources within workspaces.